salesforcecli/plugin-functions
# feat: Add sf create compute collaborator add command

### What does this PR do?
Adds a heroku user to the the functions via cli. This enables users to use any of their compute environments
with heroku addons.

## How to verify?
1. Pull down the PR branch
2. Enable the feature flag `heroku sudo labs:enable allow-functions-collab --user username@heroku.com`
2. Make sure that you have your scratch org, dev hub, compute environments all setup
3. Attach the compute environment to one of your databases locally by running this command from within the branch directory `sf env compute collaborator add --heroku-user:HEROKU_USER`
4. Check to see if you received an email like this

<img width="510" alt="Screen Shot 2022-02-24 at 9 17 11 AM" src="https://user-images.githubusercontent.com/22944735/155575676-72b483c1-5b07-4924-b9e9-5438e03c1a49.png">

5. Try and add yourself again and see if you get this error
`Collaborator <heroku-user> has already been added.`
6. Try and add a heroku user account that doesn't exist and see if you get this error
`There is no Heroku User under the username <heroku-user>.`
7. Try and add a heroku user without the `-h` flag and see if you get this error
`Missing required flag: -c, --heroku-user heroku-user  <heroku-user> See more help with --help`
